regexes/client/mobile_apps.yml

```diff
@@ -271,3 +271,42 @@
   regex: 'Procast/?([\d\.]+)?'
   name: 'Procast'
   version: '$1'
+-
+  regex: 'TikTok/(\d+[\.\d]+) CFNetwork'
+  name: 'TikTok'
+  version: ''
```

Comment:
Isn't TikTok (and maybe others) available for Android as well? Those regexes would only match the iOS versions...

Reply:
I do not have a user agent for android

Reply:
Don't have it either. Maybe we could simply remove the `CFNetwork` at least for those apps we now there is an Android version maybe?

Reply:
I removed the prefix `CFNetwork` for tiktok